src/utils/getLocaleStorage.js

```diff
@@ -0,0 +1,13 @@
+export const getLocaleStorage = (key) => {
+  const value = window.localStorage.getItem(key);
```

Comment:
It's a good practice to check if the `localStorage` is available before using it. This is because some browsers may not support `localStorage`, or the user may have disabled it. You can do this by checking `if (window.localStorage) {...}` before trying to get the item.